### PR TITLE
Added a tooltip to the WHO region filter

### LINKED ISSUE #232: Define the WHO region acronyms somewhere
We need to define the WHO region acronyms somewhere for people who aren't familiar with them. Maybe a note above the visualizations?

AFR: African Region
AMR: Region of the Americas
EMR: Eastern Mediterranean Region
EUR: Region of the Americas
SEAR: South East Asia Region
WPR: Western Pacific Region

### PR DESCRIPTION
**Pull request by SeanKennyNF**
Resolves #232. Added a tooltip to the WHO region filter

Peek 2024-03-24 10-56

### CODE REVIEW

File: app/pathogen/arbovirus/dashboard/filters.tsx [hunk 1]
```diff
@@ -51,14 +58,54 @@ const addFilterMulti = (
...
+            </div>
+          </TooltipTrigger>
+          <TooltipContent
+            style={{
+              position: "absolute",
+              top: "50px", // position below the trigger
+              left:"-120px",
+              minWidth: "230px",
+              paddingTop: 0,
+              paddingBottom: 0,
```
Comment (SeanKennyNF) [inline app/pathogen/arbovirus/dashboard/filters.tsx:85]: Removes the awkward transparent box around the tooltip

File: app/pathogen/arbovirus/dashboard/filters.tsx [hunk 2]
```diff
@@ -51,14 +58,54 @@ const addFilterMulti = (
...
 
+interface FieldTooltipProps {
+  tooltipContent: React.ReactNode,
+  className?: string,
+}
+
+const FilterTooltip = (props: FieldTooltipProps): React.ReactNode => {
+  return (
+    <div className={props.className}>
+      <TooltipProvider delayDuration={0}>
```
Comment (SeanKennyNF) [inline app/pathogen/arbovirus/dashboard/filters.tsx:69]: Appears instantly when moused over

File: components/customs/SectionHeader.tsx [hunk 3]
```diff
@@ -17,43 +17,35 @@ const SectionHeader: React.FC<SectionHeaderProps> = ({
   header_text,
   tooltip_text,
 }) => {
-  const [showTooltip, setShowTooltip] = useState(false);
```
Comment (SeanKennyNF) [inline components/customs/SectionHeader.tsx:20]: Code could be simplified a fair amount

File: components/customs/SectionHeader.tsx [hunk 4]
```diff
@@ -17,43 +17,35 @@ const SectionHeader: React.FC<SectionHeaderProps> = ({
...
-                  minWidth: "230px", // Set a minimum width
-                }}
+            <TooltipContent
+              style={{
+                position: "absolute",
+                top: "50px", // position below the trigger
+                left:"-120px",
+                minWidth: "230px", // Set a minimum width
+                paddingTop: 0,
+                paddingBottom: 0,
```
Comment (SeanKennyNF) [inline components/customs/SectionHeader.tsx:40]: Same thing, removes that awkward box

File: components/customs/SectionHeader.tsx [hunk 5]
```diff
@@ -17,43 +17,35 @@ const SectionHeader: React.FC<SectionHeaderProps> = ({
...
 }) => {
-  const [showTooltip, setShowTooltip] = useState(false);
-
   return (
     <div className="flex items-center space-x-2">
       <h2 className="text-lg">{header_text}</h2>
       <div className="relative inline-block">
-        {/* Tooltip */}
-        <TooltipProvider>
+        <TooltipProvider delayDuration={0}>
```
Comment (SeanKennyNF) [inline components/customs/SectionHeader.tsx:24]: Better UX imo

**Review by EliasPinno — APPROVED** [review]

**Review by EliasPinno — APPROVED** [review]

**Pull request merged**